.github/workflows/build.yml

```
@@ -637,9 +637,9 @@ jobs:
           ssh-keyscan "${SSH_HOST}" >> "${HOME}/.ssh/known_hosts"
           echo "SSH_AUTH_SOCK=${SSH_AUTH_SOCK}" >> "${GITHUB_ENV}"
 
-      - name: "[macOS/Windows] Upload build to downloads.mixxx.org"
+      - name: "[Android/macOS/Windows] Upload build to downloads.mixxx.org"
         # skip deploying Ubuntu builds to downloads.mixxx.org because these are deployed to the PPA
-        if: runner.os != 'Linux' && inputs.publish && env.SSH_AUTH_SOCK != null
+        if: not(startsWith(matrix.artifacts_slug, 'ubuntu')) && inputs.publish && env.SSH_AUTH_SOCK != null
```

Looks like `not` is not a function (See [CI error](https://github.com/mixxxdj/mixxx/actions/runs/19613417351))

> Unrecognized function: 'not'.

```suggestion
        if: !startsWith(matrix.artifacts_slug, 'ubuntu') && inputs.publish && env.SSH_AUTH_SOCK != null
```